### DISCUSSION TITLE
定制菜单栏图标问题

### DISCUSSION THREAD

**Discussion by ChenKun1997** [wangeditor-team/wangEditor#4546] [General]
我只想更换菜单栏图标，但是现在好像只有菜单组可以自定义图标，但是我又不想用菜单组，因为我没有多个功能放到一个组里面，菜单组是多此一举，那么非菜单组怎么自定义图标呢

**Comment by wangfupeng1988** [comment]
为啥要更换菜单栏图标？哪个图标不合适？

**Reply by ChenKun1997** [reply]
> 为啥要更换菜单栏图标？哪个图标不合适？

没办法，公司UI验收的时候让我换成她选的图标，不是合适不合适的问题

**Comment by wangfupeng1988** [comment]
好，我记录下这个需求

排队开发
